pandas/core/series.py

```diff
@@ -3614,6 +3614,9 @@ def _reduce(self, op, name, axis=0, skipna=True, numeric_only=None,
         elif is_datetime64_dtype(delegate):
             # use DatetimeIndex implementation to handle skipna correctly
             delegate = DatetimeIndex(delegate)
+        elif is_timedelta64_dtype(delegate) and hasattr(TimedeltaIndex, name):
```

Comment:
why would you need to check if the method exists we don't do this for any other types

Reply:
otherwise this breaks for e.g. `std`

Reply:
but it doesn't break for datetime64, so puzzled why this is needed here

Reply:
?

Reply:
not sure what you're asking for here, buddy.

Reply:
eel my comment is why are you checking if the method exists when we don’t use this anywhere else

Reply:
because failing to check for this causes a test failure.  In particular, for `std`, where a timedelta64 ndarray has the method but TDA doesn't yet

Reply:
the is true for DTA so your answer doesn’t make sense

Reply:
`std` is _supposed_ to raise for datetime64 in the status quo, so we don't test it

Reply:
still this is just smelly. I don't see a problem delegating to TDI here if its a timedelta type. there must be something else wrong if you need this check.